projects.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: page
+title: Current Projects
+descriptiveTitle: Toronto Mesh Projects
+order: 1
+parent: Get Involved
+---
+
+---
+Node Technology and Mesh Applications
+
+---
+
+### Meshstream
+
+Meshstream is a collection of software and hardware that aims to show what can be accomplished with peer-to-peer applications running over a wireless mesh network that is completely isolated from the Internet.
+
+Meshstream demonstrates:
+- Live video streaming over content addressable storage ([IPFS](https://ipfs.io/))
+- Sharing multimedia content over a peer-to-peer social network ([SSB](https://github.com/ssbc/secure-scuttlebutt))
+- Mesh networking over long-range wireless links using open hardware ([LibreRouter](https://librerouter.org/))
+
+**Status**: Alpha  
+Demo-quality software tested on LibreRouter prototype hardware.
+
+**Skills**  
+Flashing routers, OpenWrt, network configuration, programming peer-to-peer applications, shell scripts, web applications, web sockets, javascript, single-board computers, technical documentations, web design, user story design.
+
+[Meshtream GitHub page](https://github.com/tomeshnet/meshstream)
```

Comment:
Use the snazzy gh icon?

`{% include icon-github.html %}`

Reply:
I did not know about this! They are snazzy indeed :)